### PR TITLE
feat: added text overrides on CartItem

### PR DESCRIPTION
**Pull request by HarisSpahijaPon**
Part of: #409
Impact: minor
Type: feature

Component
CartItem now has overrides when i18next will be implemented (see #409)

Breaking changes
No breaking chances, all added fields are optional

Testing
Add a new prop `messageText` or `totalText` to `<CartItem />`
Add the value "test" to either `messageText` or `totalText`
Button or title should display "test" when test is given as a value to the prop. When the prop `messageText` or `totalText` is not added, it should default to the values given in default props

### CODE REVIEW

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 2d740214a667f121563e432ad06bad9a34c323eb

https://deploy-preview-449--stoic-hodgkin-c0179e.netlify.com

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 7889739c52faaecc9811f89213fd255db2dc86f1

https://deploy-preview-449--stoic-hodgkin-c0179e.netlify.com

File: package/src/components/CartItem/v1/CartItem.js [hunk 1]
```diff
@@ -265,14 +265,28 @@ class CartItem extends Component {
...
-    productURLPath: PropTypes.string
+    productURLPath: PropTypes.string,
+    /**
+     * Text to display inside the remove button
+     */
+    removeText: PropTypes.string,
+    /**
+     * The text for the "Remove" button text.
+     */
+    removeText: PropTypes.string,
```
Comment (aldeed) [inline package/src/components/CartItem/v1/CartItem.js]: Remove duplicate `removeText` prop type

Comment (HarisSpahijaPon) [reply package/src/components/CartItem/v1/CartItem.js]: done

**Pull request merged**